dor/adapters/technical_metadata.py

```diff
@@ -48,8 +56,42 @@ class ImageTechnicalMetadata:
     def create(cls, file_path: Path) -> Self:
         jhove_doc = JHOVEDoc.create(
             file_path,
-            JHOVE_IMAGE_METADATA_PROPERTY
+            cls.metadata_property()
+        )
+
+        mimetype = jhove_doc.mimetype
+        if mimetype.startswith("image/"):
+            cls_ = ImageTechnicalMetadata
+        elif mimetype.startswith("text/"):
+            cls_ = TextTechnicalMetadata
+        else:
+            cls_ = cls
+
+        # set the metadata_property
+        jhove_doc.metadata_property = cls_.metadata_property()
```

Comment:
Why do we need to set this? Isn't it already set by passing `cls.metadata_property()` on line 49?

Reply:
Because `TechnicalMetadata.create` doesn't have the correct `metadata_property` to pass to `JHOVEDoc` at this point.